AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -1627,7 +1628,6 @@ ratemyprofessors.com###cookie_notice
 ratemyprofessors.com##.preventAcceptance.balaMainContainer
 edeka.de##.o-core-cookie-notification
 bosch.de##o-cookie-law
-siemens.com##.nts-notification-cookie
```

Comment:
If I remember, you changed that siemens.com already in another pull request, so please remove the remove here.